src/js/angular/clustermanagement/templates/modal/cluster-edit-dialog.html

```diff
@@ -80,6 +80,18 @@ <h3 class="modal-title">{{pageTitle}}</h3>
                             </div>
                         </div>
                     </div>
+
+                    <div class="form-group">
+                        <div class="input-group" ng-class="{'has-danger': clusterConfigurationForm.batchUpdateInterval.$invalid}">
+                            <label for="batch-update-interval" class="col-xs-4 col-form-label">{{'cluster_management.cluster_configuration_properties.batch_update_interval' | translate}}*</label>
+                            <div class="col-xs-8" gdb-tooltip="{{'cluster_management.cluster_configuration_properties.batch_update_interval_tooltip' | translate}}">
+                                <input class="form-control" placeholder="{{'required.field' | translate}}" type="number" ng-model="clusterConfiguration.batchUpdateInterval"
+                                       required id="batch-update-interval" min="0" pattern="[0-9]+" name="batchUpdateInterval">
+                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.batchUpdateInterval.$error.required">{{'required.field' | translate}}</div>
+                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.batchUpdateInterval.$error.pattern">{{'cluster_management.cluster_page.errors.only_positive_integers' | translate}}</div>
```

Comment:
I wonder. Given the fact that the `batch-update-interval` field (which is incorrect name for an id BTW), is of type=number, and you declarred it as min=0, then why do you need the pattern and this validation? IMO it's just redindant.

Reply:
you are right - refactored the whole html file